extensions/geode-modules/src/integrationTest/java/org/apache/geode/modules/session/Tomcat6SessionsJUnitTest.java

```diff
@@ -20,10 +20,10 @@
 import org.apache.geode.modules.session.catalina.Tomcat6DeltaSessionManager;
 import org.apache.geode.test.junit.categories.SessionTest;
 
-@Category({SessionTest.class})
+@Category(SessionTest.class)
+// TODO: Remove the entire class once Tomcat6DeltaSessionManager is removed from the product.
```

Comment:
Generally I am anti-TODO because they tend to just languish in the code, but in this case it is a nice comment because it indicates that this code should really be deprecated.